cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -402,6 +408,11 @@ public void init() {
                     logger.atInfo().withCause(e).log(re.toString());
                     ctx.status(HttpServletResponse.SC_NOT_FOUND).json(re);
                 })
+                .exception(RateException.class, (e, ctx) -> {
```

Comment:
There is no parent class that RateException can extend?  Otherwise what keeps ApiServlet from filling up with implementation specific exception types?  I would think that if we really can't think of anything more generic at the very least we should be able to make a class for Controller exceptions and another for DaoExceptions depending on where the issue is.  I don't see ApiServlet knowing how to handle RateException in any helpful or meaningfully different way than other exceptions.

Reply:
Though I didn't see it before. I completely agree with Ryan on this one.

Though based on the suggestion of DAO vs Controller exception, would it be better to accept for now and create a follow up issue to review the exception handling.

This particular block has already been getting a bit large and it's possible that just a few marker interfaces may help sort that out.

Reply:
I agree, I'd like to move that refactor into a subsequent effort as my ideas for implementing something standard are either insufficient or too large of a change for this PR.